blazor/datagrid/selection.md

```diff
@@ -732,6 +732,68 @@ In the following sample, the third row is initially selected using the [Selected
 
 N> You can refer to our [Blazor DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid) feature tour page for its groundbreaking feature representations. You can also explore our [Blazor DataGrid example](https://blazor.syncfusion.com/demos/datagrid/overview?theme=bootstrap4) to understand how to present and manipulate data.
 
+## Prevent the selection of records in the DataGrid after selecting a n number of records
+
+The Syncfusion Grid component provide support for single and multiple selection in Grid. By default, users can choose between single and multiple selection modes by utilizing the [Type](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridSelectionSettings.html#Syncfusion_Blazor_Grids_GridSelectionSettings_Type) property of **GridSelectionSettings**. However, in multiple selection modes, to restrict the selection after selecting a n number of records. This can be achieved by utilizing the [RowSelecting](https://blazor.syncfusion.com/documentation/datagrid/events#rowselecting) event of the Grid.
```

Comment:
However, in multiple selection modes, to restrict the selection after selecting a n number of records -----> has no meaning. can be phrased as "However, in multiple selection modes, you can restrict the selection after selecting a n number of records in DataGrid component"